compiler-rt/include/fuzzer/FuzzedDataProvider.h

```diff
@@ -13,6 +13,8 @@
 #ifndef LLVM_FUZZER_FUZZED_DATA_PROVIDER_H_
 #define LLVM_FUZZER_FUZZED_DATA_PROVIDER_H_
 
+#include <stdlib.h>
```

Comment:
We should be using `cstdlib` instead, see the other `c*` header inclusions below.